datastore/cloud-client/snippets.py

```diff
@@ -66,6 +186,9 @@ def basic_entity(client):
             "description": "Learn Cloud Datastore",
         }
     )
+    # Access fields
+    task["category"]  # "Personal"
+    task["priority"]  # 4
```

Comment:
I find this confusing. I get showing how to access a field, but statements that are just a value aren't something I'd expect. You might want to consider doing something with these values, such as logging them, to show not only how to reference them but also how to use the result.